libpod/options.go

```diff
@@ -305,6 +305,23 @@ func WithSELinuxLabels(processLabel, mountLabel string) CtrCreateOption {
 	}
 }
 
+// WithUser sets the user/group to run with the container
+func WithUser(user string) CtrCreateOption {
+	return func(ctr *Container) error {
+		if ctr.valid {
+			return ErrCtrFinalized
+		}
+
+		uid, gid, err := ctr.ParseUser(user)
```

Comment:
I don't know if we guarantee the container is mounted when this runs. Might be safer to just store a bool UseImageUser and do the ParseUser when we create the spec

Reply:
Moved, my only concern is that the spec is always created so that the value is checked, but I think we are ok.